🚚 update: add 128th CatchUp Summary (#152)
OurTechCommunity/catchup: summary/sessions/128/content.adoc

Date: 22-04-2023

Duration: 6 hrs 15 mins

==== Topics Discussed

> * General Introductions.

Reviewer: ```suggestion
* General introductions.
```

* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] talked about an issue that he encountered while building Homa Module in Ubuntu.
    ** The issue: link:https://github.com/PlatformLab/HomaModule/issues/30[Build error: `cpu_khz` undeclared^].
* link:https://twitter.com/WilfredAlmeida_[Wilfred Almeida^] showcased his link:https://wilfredalmeida.com/resume[Resume^] that he had self-hosted on a Debian-11 instance on a remote workstation on link:https://www.hetzner.com[Hetzner^] for learning purposes.
    ** He embedded the PDF in the HTML with tags, which results in a cleaner URL.
    ** We discussed the "/" problem, where a URL might not be redirected to the correct location sometimes. 

>         *** Generally Flask and Django applications are affected.

Reviewer: ```suggestion
        *** Generally link:https://palletsprojects.com/p/flask[Flask^] and link:https://djangoproject.com[Django^] applications are affected.
```

        *** link:https://flask.palletsprojects.com/en/2.2.x/quickstart/#unique-urls-redirection-behavior[Unique URLs / Redirection Behavior^]
* link:https://twitter.com/tusharnankanii[Tushar Nankani^] shared a link:https://www.alumni.hbs.edu/Documents/careers/ActionVerbsList.pdf[Action Verbs List^] and a guide for writing link:https://www.alumni.hbs.edu/careers/job-search/Pages/resumes-and-cover-letters.aspx[Resume and cover letter^] that he found on the Harvard Business School website.
* We had a short discussion about the skewed gender ratio in colleges and the tech industry which results in interesting observations such as:

>     ** The gender with lower representation is often given more opportunities to make it neutral. (Controversial)

Reviewer: ```suggestion
    ** The gender with lower representation is often given more opportunities (equity) to improve the gender ratio (equality).
```

* We discussed how some colleges or universities carry a certain prestige with them which gives an edge to the students during recruitment.
* link:https://twitter.com/ambitions2003[Siddharth Kaduskar^] asked why some people prefer finding friends or partners while solving CP (Competetive Programming) or for hackathons.
    ** link:https://twitter.com/adityaoberai1[Aditya Oberai^] suggested that most such groups are meant as accountability partners.
    ** link:https://twitter.com/SirusTweets[Darshan Rander^] offered link:https://twitter.com/ambitions2003[Siddharth Kaduskar^] to work on additional features for link:https://github.com/SirusCodes/Heartry/issues[Heartry^] to evaluate if he wants to work with someone or not.
* We talked about how one can find good partners for Hackathons. 
    ** link:https://twitter.com/skxrxn[Sreekaran Srinath^] and link:https://twitter.com/adityaoberai1[Aditya Oberai^] shared their experiences from link:https://mlh.io[MLH^] and individual indie hacking.

> * We had a chat about credit cards (Credit card Churning) and the perks that come with good credit scores and better cards.

Reviewer: ```suggestion
* We had a chat about credit cards (link:https://www.discover.com/credit-cards/card-smarts/what-is-credit-card-churning[Credit Card Churning^]) and the perks that come with good credit scores and better cards.
```

* Apple stores launched in India are well equipped for the experience but seems like the policies differ quite a bit from its stores in the US.
    ** The function of these stores mostly seems to align with prestige and trust.
* link:https://twitter.com/Aryannayakk[Aryan Nayak^] talked about his routine for solving link:https://leetcode.com[Leetcode problems^].
    ** We talked about health, gym and fitness. This conversation culminated in a discussion about insulin resistance and how strenuous activities might eat up the insulin in the blood which also reduces the risk of diabetes.

>     ** He shared a book that he was currently reading: link:https://www.amazon.co.uk/Outlive-Science-Longevity-Peter-Attia/dp/1785044540/ref=asc_df_1785044540/?tag=googshopuk-21&linkCode=df0&hvadid=606653503569&hvpos=&hvnetw=g&hvrand=5203389675007555026&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9044962&hvtargid=pla-1888518939961&psc=1&th=1&psc=1[Outlive: The Science and Art of Longevity Hardcover^]

Reviewer: ```suggestion
    ** He shared a book that he was currently reading: link:https://www.amazon.co.uk/Outlive-Science-Longevity-Peter-Attia/dp/1785044540[Outlive: The Science and Art of Longevity^]
```
